src/datetime-picker.component.ts

```diff
@@ -140,6 +140,15 @@ declare var moment: any;
   `,
   styles: [
     `
+* {
```

Comment:
We are using no encapsulation as you see in the code to let people override css.
```
  encapsulation: ViewEncapsulation.None
```

My question is why use `content-box`, instead of `border-box`?